controllers/provider-azure/charts/internal/cloud-provider-config/templates/cloud-provider-config.tpl

```diff
@@ -21,9 +21,9 @@ cloudProviderBackoffExponent: 1.5
 cloudProviderBackoffDuration: 5
 cloudProviderBackoffJitter: 1.0
 cloudProviderRateLimit: true
-cloudProviderRateLimitQPS: 10.0
+cloudProviderRateLimitQPS: {{ printf "%d%s" ( add .Values.maxNodes 10 ) ".0" }}
 cloudProviderRateLimitBucket: 100
-cloudProviderRateLimitQPSWrite: 10.0
+cloudProviderRateLimitQPSWrite: {{ printf "%d%s" ( add .Values.maxNodes 10 ) ".0" }}
```

Comment:
Hmm, do you always add 10 to maxNode? Isn't better to use maxNodes if maxNodes >= 10, else - to use 10 as minimum.

Reply:
See https://github.com/kubernetes-sigs/cloud-provider-azure/issues/155#issuecomment-488523167.

Reply:
Nice reference 👍 Thanks, makes sense. I agree with @ialidzhikov suggestion (`max{10, maxNodes}`).

Reply:
Done.
I also removed trailing `.0` as it seems it is not required. See this example [here](https://github.com/kubernetes-sigs/cloud-provider-azure/blob/fb507c32b206a110f4c8bbf4316c7417101ce3ef/examples/azure.json#L23).